Zero-shot-Instance-Segmentation/README.md

```diff
@@ -0,0 +1,154 @@
+
+# Code for CVPR2021 paper
+
+# **Zero-shot Instance Segmentation** 
+
+## Code requirements
++ python: python3.7
++ nvidia GPU
++ pytorch1.1.0
++ GCC >=5.4
++ NCCL 2
++ the other python libs in requirement.txt
+
+## Install 
+
+```
+conda create -n zsi python=3.7 -y
+conda activate zsi
+
+conda install pytorch=1.1.0 torchvision=0.3.0 cudatoolkit=10.0 -c pytorch
+
+pip install cython && pip --no-cache-dir install -r requirements.txt
+   
+python setup.py develop
+```
+
+## Dataset prepare
+
+
+- Download the train and test annotations files for zsi from [annotations](https://dagshub.com/f2010126/Zero-shot-Instance-Segmentation/src/main/data/coco/annotations), put all json label file to
+    ```
+    data/coco/annotations/
+    ```
+
+- Download MSCOCO-2014 dataset and unzip the images it to path： 
+    ```
+    data/coco/train2014/
+    data/coco/val2014/
+    ```

```

Comment:
1. There is no `val2014` folder in the project, only `train2014` and `test2014`, so I'm not sure if we need to change the name of the folder here or in the project, but they are currently incompatible.
2. The data isn't uploaded to DagsHub even though it seems you locally did track it, would it be possible to push it in, so that these URLs work: https://dagshub.com/f2010126/Zero-shot-Instance-Segmentation/src/main/data/coco/test2014 and https://dagshub.com/f2010126/Zero-shot-Instance-Segmentation/src/main/data/coco/train2014.